multi-QV-with-Polling.nlogo

```diff
@@ -221,6 +223,29 @@ to set-utilities
       ]
     ]
   )
+  if correlate-issues != 0 [
+    let first-issue-utility item 0 utilities
+    let temp-utilities (list)
+    let curr-issue 0
+    ; Adding in a negative correlationas well (in other words, the issues selected will have a negative correlation with the first issue), this is less relevant from an analytic perspective, but I left it in just in case.
+    repeat correlate-n-issues [
+      ifelse correlate-issues > 0 [
```

Comment:
I don't like the name "correlate-issues" for the issue correlation. It sounds like a command. Maybe call it issues-correlation